### ISSUE TITLE
KS3 and over-clustering?

### ISSUE THREAD

**Issue by DemetrisS** [MouseLand/Kilosort#409]
Hello

I am using KS3 with data from a 24Ch linear probe (Uprobe) - I am ending with >120 clusters (some cases >200) with around  ~50% of them classified as 'good'  - this seems rather high for the number of channels and can require a fair bit of manual merging afterwards. 

I have tried the suggestions from issue #67 -  changing "ops.Nfilt" and AUCsplit. I have played around by using a wide range of values from 0.5 to 0.99 for the AUCsplit but this has very little effect on cluster numbers. 

I am not sure if this is related, but in order to get KS3 to work I had to set the "ops.nblocks" in the config file to zero.

Any advice/suggestions would be most welcome.

Many thanks
D

**Comment by JonathanAMichaels** [comment]
I don't believe the AUCsplit does anything in KS3. Making nblock=0 means no drift correction is taking place.
One possibility is to manually change the thresholds for merging in find_merges. Merging clusters with similar templates here could save you time later.

**Comment by DemetrisS** [comment]
Many thanks for your suggestion - will give it a go.

Best,
D
